applications/launchpad_v2/src/styles/themes/light.ts

```diff
@@ -28,6 +35,15 @@ const lightTheme = {
     accentDark: styles.colors.tari.purpleDark,
     disabledText: styles.colors.dark.placeholder,
     tariGradient: styles.gradients.tari,
+    info: styles.colors.secondary.info,
+    infoText: styles.colors.secondary.infoText,
+    on: styles.colors.secondary.on,
+    onText: styles.colors.secondary.onText,
+    onTextLight: styles.colors.secondary.onTextLight,
+    warning: styles.colors.secondary.warning,
+    warningText: styles.colors.secondary.warningText,
+    expert: 'rgba(147, 48, 255, 0.05)',
+    expertText: styles.gradients.tari,
```

Comment:
You can add all newly added fields to the `dark.ts` as well. Do not worry about values, just copy-paste to have the same structure in both files.

Reply:
👍